content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1790,6 +1790,27 @@ uses this annotation.
 See [annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
 in the AWS load balancer controller documentation.
 
+### service.beta.kubernetes.io/aws-load-balancer-security-groups (deprecated) {#service-beta-kubernetes-io-aws-load-balancer-security-groups}
+
+Example: `service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f, sg-8725gr62r"`
+
+Used on: Service
+
+The AWS load balancer controller uses this annotation to specify a comma seperated list
+of frontend securityGroups you want to attach to an NLB. Both name and ID of securityGroups
+are supported where name matches a `Name` tag, not the `groupName` attribute.
+
+When this annotation is added to the Service, the controller attaches the security groups
```

Comment:
```suggestion
When this annotation is added to a Service, the load-balancer controller attaches the security groups
```

Reply:
Right. that might clarify in case of confusion. will add this and update.